integration-tests/src/test/java/gov/uk/di/ipv/cri/common/api/stepDefinitions/APISteps.java

```diff
@@ -40,6 +42,21 @@ public void setAuthorizationJARForTestUser(int rowNumber)
         sessionRequestBody = IpvCoreStubUtil.sendCreateSessionRequest(userIdentityJson);
     }
 
+    @Given("Session lambda implementation is in {string}")
+    public void setSessionEndpoint(String endPoint) {
+        if (endPoint.equals("TS")) DEV_SESSION_URI = ENVIRONMENT + "/session-two";
```

Comment:
Could we add some logging to the other methods, so we can be sure that we're hitting the expected endpoints?

If this code doesn't work for some reason, it will fall back to the existing Java endpoints.